modules/Bio/EnsEMBL/Compara/RunnableDB/LoadMembers/CompareNonReusedGenomeList.pm

```diff
@@ -0,0 +1,120 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::LoadMembers::CompareNonReusedGenomeList
+
+=head1 SYNOPSIS
+
+    standaloneJob.pl Bio::EnsEMBL::Compara::RunnableDB::LoadMembers::CompareNonReusedGenomeList \
+        -compara_db $(mysql-ens-compara-prod-8 details url jalvarez_vertebrates_load_members_99) \
+        -expected_updates_file <(echo "homo_sapiens"; echo "mus_musculus"; echo "equus_caballus") \
+        -current_release 99 -nonreuse_ss_id 10000002
```

Comment:
I didn't know runnables should have this synopsis format... I like it :)
But I would replace `SYNOPSIS` by `EXAMPLE` in this case, as I think the synopsis shouldn't have specific values, e.g. the release number.

Reply:
Will use `EXAMPLE` instead. With the `=item` list below I think it will be clear how to invoke the runnable